### PR TITLE
Rake タスク claim の改良

### LINKED ISSUE #36: claim.rake の改良
**現状:**
```ruby
task :release_stale, [:older_than_hours, :dry] => :environment do |_task, args|
  older_than_hours = args[:older_than_hours]&.to_f || 1.0
  dry_run = truthy?(args[:dry])
  service = Verbena::MailQueuesService.new
  stale_count = service.release_stale_claims(older_than_hours: older_than_hours, dry_run: dry_run)
  # ...
end
```

**問題点:**
- ⚠️ エラー時のハンドリングがない（Service 層のエラーが素通り）
- ⚠️ 引数の型変換エラー時の挙動が不明確
- ⚠️ テストなし

**影響度:** 中（定期的なメンテナンス作業で使用）

### PR DESCRIPTION
**Pull request by hiroaki**
#12 の課題のうちの #36 の対応です。

This pull request improves the robustness and usability of the `verbena:claim` Rake tasks and the underlying `MailQueuesService` by adding strict argument validation, better error handling, and more comprehensive test coverage. The changes ensure that invalid or unexpected input is handled gracefully, with clear error messages and consistent behavior across both service methods and Rake tasks.

**Validation and error handling improvements:**

* Added a new `normalize_hours_arg` method to `MailQueuesService` that validates and normalizes the `older_than_hours` argument, ensuring it is a non-negative number and providing clear error messages for invalid input. This method is now used throughout the service and Rake tasks.
* Updated the `release_stale_claims` method to use the normalized hours value for logging and processing, ensuring consistency in logs and behavior. [[1]](diffhunk://#diff-ef9a76f88490818e31b374f712906a6448e28cac2c67e842443861fbe4f6df6fL97-R98) [[2]](diffhunk://#diff-ef9a76f88490818e31b374f712906a6448e28cac2c67e842443861fbe4f6df6fL108-R109) [[3]](diffhunk://#diff-ef9a76f88490818e31b374f712906a6448e28cac2c67e842443861fbe4f6df6fL118-R119)
* Enhanced the `verbena:claim:release_stale` and `verbena:claim:show_stale` Rake tasks to catch exceptions, print clear error messages to stderr, and exit with a non-zero status on failure. [[1]](diffhunk://#diff-286d7914d2c9df59708fe85ab4250317863897d95f968e7d60716255deb77348L5-R26) [[2]](diffhunk://#diff-286d7914d2c9df59708fe85ab4250317863897d95f968e7d60716255deb77348R43-R51)

**Testing improvements:**

* Expanded the service spec (`mail_queues_service_spec.rb`) to cover argument validation, including tests for numeric strings, nil values, negative numbers, and non-numeric input.
* Refactored the Rake task spec (`claim_rake_spec.rb`) to test error handling, input validation, and output for both normal and exceptional cases, using service doubles for isolation.

**Other code quality improvements:**

* Made the `BOOLEAN_TYPE` assignment idempotent to avoid redefinition warnings.
* Minor cleanup and restructuring of the Rake task spec for better clarity and maintainability.

### CODE REVIEW

File: lib/tasks/verbena/claim.rake [hunk 1]
```diff
@@ -2,48 +2,57 @@ namespace :verbena do
...
   end
 
   # Rake helper methods (local to this file)
-  BOOLEAN_TYPE = ActiveModel::Type::Boolean.new
+  BOOLEAN_TYPE ||= ActiveModel::Type::Boolean.new
 
   def truthy?(val)
     # Rails-native boolean casting: true => 1,true,t,on,yes | false => 0,false,f,off,no
-    BOOLEAN_TYPE.cast(val)
+    !!BOOLEAN_TYPE.cast(val)
```
[the remark this answers is not in the document]
Comment (hiroaki) [reply lib/tasks/verbena/claim.rake:55]: true / false を返したいので、これは妥当な変換です。 もし nil も返る可能性があるなら、メソッド名を変更すべきです。
